### ISSUE TITLE
Can't set linaria configuration inside babel.config.js

### ISSUE THREAD

**Issue by georgiosgiatsidis** [callstack/linaria#546]
## Environment

This is my babel.config.js
```js
module.exports = function(api) {
    let presets = [];
    let plugins = [];

    if (api.env('production')) {
        presets = [
            '@babel/preset-env',
            '@babel/preset-react',
            [
                'linaria/babel',
                {
                    displayName: false
                }
            ]
        ];
        plugins = [
            //
        ];
    } else if (api.env('development')) {
        presets = [
            '@babel/preset-env',
            '@babel/preset-react',
            [
                'linaria/babel',
                {
                    displayName: true
                }
            ]
        ];
        plugins = [
            //
        ];
    }

    return {
        presets,
        plugins
    };
};
```

## Description

I try to pass the linaria config as second argument inside babel presets instead of using a `linaria.config.js`.
The expected behavior is to see different `displayName` value for each enviroment, but the default options are used.

## Reproducible Demo

Just pass linaria configuration overrides at `babel.config.js`

**Comment by Jayphen** [comment]
Is there a particular reason you don't want to use `linaria.config.js`?

You can alternatively put the option in your webpack config:

```
      use: [
        {
          loader: "linaria/loader",
          options: {
            displayName: process.env.NODE_ENV !== "production"
          }
        }
      ]
```

**Comment by georgiosgiatsidis** [comment]
Thanks for your answer. We provide  the mode configuration option to production/development in webpack config so we don't need to set a redundant  environment variable in our package.json. So linaria.config.js does not have the environment. Your solution seems to works fine.

I was just wondering if you know why cannot pass the configuration in the babel.config.js. I debugged the extract function in`src/babel/extract.ts` and I discovered that my overrides are passed in the first react file and then in the rest files the default options are used.
